src/cljs/nr/deckbuilder.cljs

```diff
@@ -203,7 +205,7 @@
         (->> @all-cards
              (filter #(and (= (:side %) side)
                            (= (:type %) "Identity")))
-             (filter #(not (contains? %1 :replaced_by))))
+             (remove :rotated))
```

Comment:
I think this should stays as `:replaced_by`, which is used to map alt arts to multiple cards of the same name (eg. in Core 2.0). I don't think you want to remove rotated cards from the lookup search.

Reply:
That's a good point. How do we want to handle cards with more than two entries, then? What does `:replaced_by` mean when it's in core 1, core 2, and core 3?

Reply:
This structure makes sense when things are linear, but less clear to me how to handle branching. The goal is not to have multiple copies of a given card name in the list (and treat the different versions of it as alternate art options). We could just leave them all in, though that might be confusing to see `Stimhack (Core Set)`, `Stimhack (Revised Core Set)`, etc.